service_files/phosphor-broadcast-active-bmc-role.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Broadcast if the BMC role is Active
+PartOf=obmc-bmc-active.target
+
```

Comment:
Testing with Matt, the After in this file is the one that needed to be removed, because the conf file says it's part of the active target, and runs after this broadcast service, so this service can't say to run after the active target.